app.py

```diff
@@ -1,8 +1,11 @@
 from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
```

Comment:
I would not encourage the use of flask-sqlalchemy but sqlalchemy itself as it has better compatibility with graphene-sqlalchemy for object type creation.